src/com/TTT/Board.java

```diff
@@ -8,11 +8,10 @@
     private String[] spaces;
 
     public Board(int size) {
-        if(size == 4) {
+        if (size == 4) {
             spaces = new String[]{"1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16"};
```

Comment:
Can you think of a way to remove the `if` statement here? i.e: So that if we asked you to additionally support a 5x5 board, you wouldn't need to make any extra changes?